### PR TITLE
Community

### PR DESCRIPTION
**Pull request by decaluwe**
Adding Community info page.

### CODE REVIEW

**Comment by decaluwe** [comment]
Not sure why the `definingphases` related commits got included, here - I thought that PR was merged?

**Comment by bryanwweber** [comment]
No, I think that's #7 which isn't merged yet

**Comment by bryanwweber** [comment]
The image file should be put in a directory called `files/images` in the root of the site (same level as `pages`), then the location will be `/images/filename.png` in the output

**Comment by decaluwe** [comment]
Ah, perfect - thanks!  Updating the PR, now...

File: pages/about.rst [hunk 1]
```diff
@@ -45,8 +42,137 @@ order) is:
...
+  * This forum is not directly moderated or supported by the Cantera developers
+    or Steering Committee.  While developers may periodically read or respond to
+    posts, there is no expectation of any ``official`` Cantera support through
+    this forum.
+  * Posts or questions directly relevant to Cantera usage or support should
+    still be directed to the Cantera Users' Group. Having this information in a
+    single, searchable repository is a great benefit to our users, and we do not
+    want Cantera-relevant information spread across multiple venues. 
+
+.. raw:: html
```
[the remark this answers is not in the document]
Comment (bryanwweber) [reply pages/about.rst]: Never mind, I'll fix this.

**Pull request merged**